src/components/appSwitch/AppSwitch.js

```diff
@@ -261,6 +262,18 @@ const AppSwitch = (props: LoadingProps) => {
   const { descriptors, navigation } = props
   const activeKey = navigation.state.routes[navigation.state.index].key
   const descriptor = descriptors[activeKey]
+  const [torusSDK, sdkInitialized] = useTorus()
+  useEffect(() => {
+    if (sdkInitialized) {
+      AsyncStorage.getItem('connectAccountsProviderLoginInitiated').then(async value => {
+        if (value) {
+          const result = await torusSDK.getRedirectResult()
+          AsyncStorage.setItem('torusRedirectResult', result)
+          navigation.navigate('ConnectedAccounts')
+        }
+      })
+    }
+  }, [sdkInitialized])
```

Comment:
we dont need  torussdk here, just the redirect
use the torus sdk in ConnectedAccounts